Assets/Features/Autopilot/Provider Experimental/AutopilotDebug.cs

```diff
@@ -0,0 +1,97 @@
+﻿using EdyCommonTools;
+using Perrinn424.AutopilotSystem;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AutopilotDebug : MonoBehaviour
+{
+    public Rigidbody rb;
+    public RecordedLap lap;
+
+    private NearestSegmentSearcher segmentSearcher;
```

Comment:
@Maesla , here the nearest segment searcher, is this a common library/file, or did you write this?

Reply:
Same for segment searcher as well! I am curious because I will check inside the code as well!

Reply:
Yeah @augustonetto , thanks for the reply, I realized this somewhere down below, I get it now! :) clear.